public/app/features/datasources/components/ButtonRow.tsx

```diff
@@ -1,31 +1,23 @@
 import React from 'react';
 
 import { selectors } from '@grafana/e2e-selectors';
-import { Button, LinkButton } from '@grafana/ui';
-import { contextSrv } from 'app/core/core';
-import { AccessControlAction } from 'app/types';
+import { Button } from '@grafana/ui';
 
 export interface Props {
-  exploreUrl: string;
   canSave: boolean;
   onSubmit: (event: React.MouseEvent<HTMLButtonElement>) => void;
   onTest: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
 }
 
-export function ButtonRow({ canSave, onSubmit, onTest, exploreUrl }: Props) {
-  const canExploreDataSources = contextSrv.hasPermission(AccessControlAction.DataSourcesExplore);
-
+export function ButtonRow({ canSave, onSubmit, onTest }: Props) {
   return (
     <div className="gf-form-button-row">
-      <LinkButton variant="secondary" fill="solid" href={exploreUrl} disabled={!canExploreDataSources}>
-        Explore
-      </LinkButton>
```

Comment:
Just pinging @teddyba and @mikkancso here, since the "Explore" button was added not long ago, is the decision to remove it agreed on between the teams?

Reply:
Thanks for pinging us, @leventebalogh.
I'm not aware of any common decision. What is the rationale behind removing this?

Reply:
This has been discussed month ago for the config redesign project. We replace the button with a link in the alert, and there's also a button in the header. cc @catherineymgui 

<img width="787" alt="Screenshot 2023-04-23 at 22 45 44" src="https://user-images.githubusercontent.com/112862936/234874176-9f8f1ff1-1c8c-4d9a-b212-d033319cc533.png">

Reply:
Yup, the rationale: You can only explore and create a dashboard for this data source once you have a successful health check. Thus having it as an action you can take on the same hierarchy of save and test is confusing (since you can only explore after saving and test). So the idea is that once this health check is successful, someone can perform this action. Like Kuba said, these are now showing as buttons in the header and upon a succesful health check

Reply:
Thanks for the explanation! Good stuff!